.github/workflows/main.yml

```diff
@@ -1,12 +1,7 @@
 name: CI
 on:
-  push:
-    branches:
-    - auto
-    - try
```

Comment:
these branches weren't used, right?

Reply:
I think we used bors at one point but then stopped using it. So it's fine to remove these.